app/ts/components/ConfigureFunding.tsx

```diff
@@ -134,24 +134,55 @@ const WithdrawModal = ({ display, blockInfo, signers, appSettings, provider }: {
+		}
+		return undefined
+	})
+
 	function withdraw() {
 		waitFor(async () => {
 			if (withdrawAmount.value.amount <= 0n) throw 'Funding account\'s balance is to small to withdraw'
 			if (!signers.value.burner) throw 'No funding account found'
 			if (!provider.value) throw 'User not connected'
-			if (!appSettings.value.relayEndpoint) throw 'No Flashbots RPC provided'
 			if (!recipientAddress.value.address) throw 'No recipient provided'
 
-			const [tx] = await signBundle([{ signer: signers.value.burner, transaction: { chainId: provider.value.chainId, from: signers.value.burner.address, to: addressString(recipientAddress.value.address), value: withdrawAmount.value.amount, gasLimit: 21000, type: 2 }}], provider.value.provider, blockInfo.value, getMaxBaseFeeInFutureBlock(blockInfo.value.baseFee, 10n))
-			const payload = JSON.stringify({ jsonrpc: '2.0', method: 'eth_sendPrivateTransaction', params: [{ tx }] })
-			const flashbotsSig = `${await provider.value.authSigner.getAddress()}:${await provider.value.authSigner.signMessage(id(payload))}`
-			const request = await fetch(appSettings.value.relayEndpoint, { method: 'POST', body: payload, headers: { 'Content-Type': 'application/json', 'X-Flashbots-Signature': flashbotsSig } })
-			const response = await request.json()
-			if (response.error !== undefined && response.error !== null) {
-				throw Error(response.error.message)
+			// Worst case scenario, attempt to send via browser wallet if no NETWORK config for chainId or previos error sending to known RPC
+			if (useBrowserProvider.value === true) {
+				try {
+					const burnerWithBrowserProvider = signers.value.burner.connect(provider.value.provider)
+					const txInput = await burnerWithBrowserProvider.populateTransaction({ chainId: provider.value.chainId, from: signers.value.burner.address, to: addressString(recipientAddress.value.address), gasLimit: 21000, type: 2, value: withdrawAmount.value.amount, maxFeePerGas: withdrawAmount.value.fee / 21000n })
+					const tx = await burnerWithBrowserProvider.signTransaction(txInput)
+					const txHash = await provider.value.provider.send('eth_sendRawTransaction', [tx])
+					return txHash as string
+				} catch (error) {
+					throw error
+				}
+			}
+
+			// If user is on network that is in NETWORK, send via ethRpc
+			const chainId = provider.value.chainId.toString(10)
+			if (!(chainId in NETWORKS)) {
+				useBrowserProvider.value = true
+				throw 'Unkown network! If you have Interceptor installed and simulation mode on please switch to signing mode and try again.'
+			}
+
+			const fundingWithProvider = signers.value.burner.connect(new JsonRpcProvider(NETWORKS[chainId].rpcUrl))
+			try {
+				const tx = await fundingWithProvider.sendTransaction({ chainId: provider.value.chainId, from: signers.value.burner.address, to: addressString(recipientAddress.value.address), gasLimit: 21000, type: 2, value: withdrawAmount.value.amount, maxFeePerGas: withdrawAmount.value.fee / 21000n })
```

Comment:
hmm what is this calculation?
`maxFeePerGas: withdrawAmount.value.fee / 21000n`